File: structure.py
class best:
    def __init__(self, position, cost):
        self.position = position
        self.cost = cost

class empty_particle:
    def __init__(self, position, velocity, cost, best):
        self.position = position
        self.velocity = velocity
        self.cost = cost
        self.best = best

class globalbest:
    def __init__(self, cost, position):
        self.cost = cost
        self.position = position

File: public/js/structure.js
export class best {
    constructor(position, cost) {
        this.position = position;
        this.cost = cost;
    }
}

export class emptyparticle {
    constructor(position, velocity, cost, best) {
        this.position = position;
        this.velocity = velocity;
        this.cost = cost;
        this.best = best;
    }
}

export class globalbest {
    constructor(cost, position) {
        this.cost = cost;
        this.position = position;
    }
}
